x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/perform_bulk_action_route.ts

```diff
@@ -66,18 +56,23 @@ interface NormalizedRuleError {
   }>;
 }
 
-const normalizeErrorResponse = (errors: RuleActionError[]): NormalizedRuleError[] => {
+const normalizeErrorResponse = (
+  errors: Array<PromisePoolError<string> | PromisePoolError<RuleAlertType>>
+): NormalizedRuleError[] => {
   const errorsMap = new Map();
 
-  errors.forEach((ruleError) => {
-    const { message } = ruleError.error;
+  errors.forEach(({ error, item }) => {
+    const { message, statusCode } =
+      error instanceof Error ? transformError(error) : { message: String(error), statusCode: 500 };
+    // Promise pool error is either a rule ID string or a rule object itself
+    const rule = typeof item === 'string' ? { id: item } : { id: item.id, name: item.name };
```

Comment:
in which case pool error can be rule ID? Can you please give me an example?

Reply:
Yea, there's a typo; should be `item` instead of `error`.

Reply:
Updated the comment